src/components/Activities/ActivitiesList.tsx

```diff
@@ -0,0 +1,65 @@
+import { Activities } from "./Definition";
+import ListView from "./ListView";
+
+const builtInActivities: Activities[] = [
+  {
+    id: 1,
+    name: "Kayaking",
+    description: "Relax on a nice sunny day with a trip down the river.",
+    voteCount: 4,
+    category: "outdoor",
+  },
+
+  {
+    id: 2,
+    name: "Movies",
+    description:
+      "If the weather is bad outside, enjoy a cozy movie in a movie theater.",
+    voteCount: 2,
+    category: "indoor",
+  },
+
+  {
+    id: 3,
+    name: "Fondue",
+    description:
+      "If you enjoy cooking, try making fondue as a romantic activity.",
+    voteCount: 2,
+    category: "indoor",
+  },
+
+  {
+    id: 4,
+    name: "Kayaking",
+    description: "An outdoor activity your friends will enjoy.",
+    voteCount: 6,
+    category: "outdoor",
+  },
+
+  {
+    id: 5,
+    name: "Salsa Class",
+    description:
+      "Learn a new dance and meet new people by attending a salsa class.",
+    voteCount: 4,
+    category: "indoor",
+  },
+];
+
+const ActivityList = () => {
+  return (
+    <ListView
+      items={builtInActivities}
+      renderer={(activity) => (
+        <div>
+          <p>{activity.name}</p>
+          <p>{activity.description}</p>
+          <p>Votes: {activity.voteCount}</p>
+          <p>Category: {activity.category}</p>
+        </div>
+      )}
+    />
+  );
+};
+
+export default ActivityList;
```

Comment:
Wow, really nice job making all this work with typescript! 🎉 I was able to simplify your code a bit by defining an interface for the `ActivityList` props and passing `builtInActivities` as props there (I moved `builtInActivities` to `App.tsx` too), and then I explicitly defined the tsx to be rendered in `ActivityList` (eliminating `ListView` altogether). But I'd really prefer to hear @dargaCode's feedback and advice here on the best way to implement this since I'm not very familiar with typescript. This is impressive though! 😄

Reply:
Dang yeah I'm gonna have to learn a little bit more about Typescript before I review this fully, 😅  but I think I see that Jillian's making a generic component that can accept any list of items and then return a mapped view of them. Cool idea!